tarantool/connection.py

```diff
@@ -137,6 +153,13 @@ def close(self):
         self._socket.close()
         self._socket = None
 
+    def is_closed(self):
+        '''
+        Returns the state of the Connection instance
+        :rtype: Boolean
+        '''
+        return self._socket is None
+
```

Comment:
I'll not insist, but, to be honest, I think it would be better to extract move base connector changes like this one into a separate commit. This one is about the Database API.

Reply:
If you don't insist, I would leave it as is, but I promise to make more atomic commits in the future.